charts/artifactory-ha/latest/questions.yml

```diff
@@ -51,6 +57,107 @@ questions:
+  show_if: "nginx.enabled=true&&nginx.service.type=LoadBalancer&&ingress.enabled=false"
+  group: "Services and Load Balancing"
+- variable: nginx.tlsSecretName
+  default: ""
+  description: "Provide SSL Secret name to configure with Nginx"
+  type: string
+  label: Config Nginx SSL Secret
+  show_if: "nginx.enabled=true&&ingress.enabled=false"
+  group: "Services and Load Balancing"
+- variable: nginx.customArtifactoryConfigMap
+  default: ""
+  description: "Provide configMap name to configure Nginx with custom `artifactory.conf`"
+  type: string
+  label: ConfigMap for Nginx Artifactory Config
+  show_if: "nginx.enabled=true&&ingress.enabled=false"
+  group: "Services and Load Balancing"
+- variable: nginx.env.skipAutoConfigUpdate
+  default: "false"
+  description: "Skip Nginx Auto Configuration Update"
+  type: string
+  label: Skip Nginx Auto Configuration Update
+  show_if: "nginx.enabled=true&&ingress.enabled=false"
+  group: "Services and Load Balancing"
+- variable: nginx.persistence.enabled
+  default: false
+  description: "enable persistence storage for nginx server"
+  type: boolean
+  label: Enable Persistent Storage for Nginx Service
+  group: "Services and Load Balancing"
+  show_if: "nginx.enabled=true&&ingress.enabled=false"
+- variable: nginx.persistence.size
+  default: 5Gi
+  description: "Nginx persistent volume size"
+  type: string
+  label: Nginx Persistent Volume Size
+  group: "Services and Load Balancing"
+  show_if: "nginx.enabled=true&&nginx.persistence.enabled=true&&ingress.enabled=false"
+- variable: nginx.persistence.storageClass
+  default: ""
+  description: "If undefined or null, uses the default StorageClass. Default to null"
+  type: storageclass
+  label: Default StorageClass for Nginx
+  show_if: "nginx.enabled=true&&nginx.persistence.enabled=true&&ingress.enabled=false"
+  group: "Services and Load Balancing"
+- variable: nginx.resources.requests.cpu
+  default: "200m"
+  description: "Nginx initial cpu request"
+  type: string
+  label: Nginx Initial CPU Request
+  show_if: "nginx.enabled=true&&ingress.enabled=false"
+  group: "Services and Load Balancing"
+- variable: nginx.resources.requests.memory
+  default: "500Mi"
+  description: "Nginx initial memory request"
+  type: string
+  label: Nginx Initial Memory Request
+  show_if: "nginx.enabled=true&&ingress.enabled=false"
+  group: "Services and Load Balancing"
+- variable: nginx.resources.limits.cpu
+  default: "1"
+  description: "Nginx cpu limit"
+  type: string
+  label: Nginx CPU Limit
+  show_if: "nginx.enabled=true&&ingress.enabled=false"
+  group: "Services and Load Balancing"
+- variable: nginx.resources.limits.memory
+  default: "1Gi"
+  description: "Nginx memory limit"
+  type: string
+  label: Nginx Memory Limit
+  show_if: "nginx.enabled=true&&ingress.enabled=false"
+  group: "Services and Load Balancing"
+
+# Artifactory Storage Settings
 - variable: artifactory.persistence.enabled
   default: false
```

Comment:
the default `artifactory.persistence.enabled` should be set to true, otherwise, the app install will encounter `Pod has unbound immediate PersistentVolumeClaims` issue.

Reply:
@guangbochen Done